### PR TITLE
Uot 99685 html open

### PR DESCRIPTION
**Pull request by taylormadehdz**
html open in cards. open doesnt work locally so pointed to 10.194.9.112:8990 for testing

### CODE REVIEW

File: frontend/src/components/documentViewer/PDFViewer.js [hunk 1]
```diff
@@ -8,15 +8,17 @@ function useQuery(location) {
...
 export default function PDFViewer({location}) {
 
 	const [filename, setFilename] = React.useState(null);
 	const [prevSearchText, setPrevSearchText] = React.useState(null);
 	const [pageNumber, setPageNumber] = React.useState(null);
 	const [isClone, setIsClone] = React.useState(false);
 	const [cloneIndex, setCloneIndex] = React.useState(null);
-
-	let query = useQuery(location);
+    const [fileUrl, setFileUrl] = React.useState(null);
```
Comment (JLyons1985) [inline frontend/src/components/documentViewer/PDFViewer.js:19]: Fix the spacing here to keep things lined up please same for line 21

File: frontend/src/components/documentViewer/PDFViewer.js [hunk 2]
```diff
@@ -28,20 +30,28 @@ export default function PDFViewer({location}) {
...
 				});
+                }
 			}
 		}
-	}, [filename, prevSearchText, isClone, cloneIndex, pageNumber]);
-
-	useEffect(() => {
+	, [filename, prevSearchText, isClone, cloneIndex, pageNumber]);
+   
+    useEffect(() => {
```
Comment (JLyons1985) [inline frontend/src/components/documentViewer/PDFViewer.js:38]: spacing here as well

File: frontend/src/components/modules/policy/policyCardHandler.js [hunk 3]
```diff
@@ -944,14 +941,13 @@ const PolicyCardHandler = {
 				item,
 				searchText
 			} = props;
-			
-			return (
+            return (
```
Comment (JLyons1985) [inline frontend/src/components/modules/policy/policyCardHandler.js:944]: Spacing

**Review by JLyons1985 — CHANGES_REQUESTED** [review]
Fix all the spacing here. Lint issues are important, keeps code readable and clean. Other than that its good.

**Comment by taylormadehdz** [comment]
Screen Shot 2021-06-09 at 12 32 21 PM

**Review by JLyons1985 — APPROVED** [review]

**Pull request merged**